lnbits/core/templates/core/wallet.html

```diff
@@ -569,7 +569,10 @@ <h6 class="text-subtitle1 q-mt-none q-mb-sm">
             ></q-spinner>
           </q-form>
         </q-card>
-        <q-card v-else class="q-pa-lg q-pt-xl lnbits__dialog-card">
+        <q-card
+          v-else-if="receive.paymentReq && receive.lnurl == null"
```

Comment:
```suggestion
          v-else-if="receive.lnurl == null"
```

Reply:
what if this condition is false: `receive.lnurl == null` (`receive.lnurl != null`)?

Reply:
The invoice QR code should only fire creating an invoice, not when an invoice is parsed out of an lnurl-withdraw